client/a8c-for-agencies/sections/overview/body/next-steps/index.tsx

```diff
@@ -1,5 +1,102 @@
-const OverviewBodyNextSteps = () => {
-	return <div>Next Steps</div>;
-};
+import { Card, CircularProgressBar } from '@automattic/components';
+import { Checklist, ChecklistItem, type Task } from '@automattic/launchpad';
+import { useTranslate } from 'i18n-calypso';
+import { A4A_ONBOARDING_TOURS_PREFERENCE_NAME } from 'calypso/a8c-for-agencies/sections/onboarding-tours/constants';
+import { useDispatch, useSelector } from 'calypso/state';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import { savePreference } from 'calypso/state/preferences/actions';
+import { getAllRemotePreferences } from 'calypso/state/preferences/selectors';
 
-export default OverviewBodyNextSteps;
+import './style.scss';
+
+export default function OverviewBodyNextSteps() {
+	const dispatch = useDispatch();
+	const translate = useTranslate();
+
+	const preferences = useSelector( getAllRemotePreferences );
+
+	const checkTourCompletion = ( prefSlug: string ): boolean => {
+		if ( preferences && A4A_ONBOARDING_TOURS_PREFERENCE_NAME[ prefSlug ] ) {
+			return A4A_ONBOARDING_TOURS_PREFERENCE_NAME[ prefSlug ] in preferences;
+		}
+		return false;
+	};
+
+	const tasks: Task[] = [
+		{
+			calypso_path: '/sites?tour=sites-walkthrough',
```

Comment:
For Calypso paths, is it possible to use the constants we define in [this file](https://github.com/Automattic/wp-calypso/blob/96fd1a71faac8482c73632a03e377f3427c5ffcb/client/a8c-for-agencies/components/sidebar-menu/lib/constants.ts#L4)? This way, we only have to look into this file when updating paths.

Reply:
https://github.com/Automattic/wp-calypso/pull/88986/commits/1784752aa236e5bcea5e56a5c7036a3f8dd38f6b :+1: